### ISSUE TITLE
Term Request: Modulation

### ISSUE THREAD

**Issue by paulnmckelvey** [CommonCoreOntology/CommonCoreOntologies#579]
I would like to request/propose that the term Modulation be added to the Event Ontology in the next release.  Recommend as a SubClass of Wave Process.

SubClass Of Wave Process
SubClass Of (Anonymous Ancestor) 'process preceded by' some 'Wave Production'
SubClass Of (Anonymous Ancestor) 'has process part' some 'Wave Process Profile'

Modulation = def. A process of impressing intelligence upon a transmission medium.

Definition source = Navy Electricity and Electronics Training Series: Module 12 - Modulation.  Navy Electricity and Electronics Training Series: Module 20 - Master Glossary.

https://archive.org/download/NEETSModule12/NEETSModule12.pdf
https://archive.org/download/NEETSModule20/NEETSModule20.pdf

**Comment by paulnmckelvey** [comment]
How about two modifications: 1) adjust the entity term to "Wave Modulation," as a SubClass of "Wave Process."  And 2), change the definition to, "A process of imposing an input signal onto a carrier wave."

**Comment by alanruttenberg** [comment]
This triggers the natural vs intentional process distinction. Wave Production, Wave Process are "Natural Process": 
>A Process existing in or produced by nature; rather than by the intent of human beings.

I don't think we're talking about a natural process here. FWIW I think the distinction is difficult to make and many processes, it seems to me, can be both natural and intentional.

**Comment by alanruttenberg** [comment]
Re Natural vs Intentional see https://github.com/CommonCoreOntology/CommonCoreOntologies/discussions/398, which was (grrrr) changed to a discussion.

**Comment by paulnmckelvey** [comment]
Would this then fall under "Act"/"Planned Act" (as a synonym for "Intentional Act")?

**Comment by paulnmckelvey** [comment]
@alanruttenberg I understand where you are coming from with your assertion.  Without trying to resolve the greater Intentional Act vs. Natural Process debate, I would reference examples of "Propulsion Process" (SubClass of "Natural Process") as an analogous argument for [Electromagnetic] Wave Modulation as a natural process:

"example: 

"a twin-engine turboprop plane rotating both of its propellers against a portion of atmosphere to propel the plane forward

"launching a water balloon using a sling shot"

**Comment by alanruttenberg** [comment]
@neilotte I'm struggling to understand how an airplane flying can be considered a natural process. It seems the most planned of acts, from the design of the wings to the active control of the throttle to modulate thrust. Maybe it's just a bad example of the class? A rock flying out of a volcano is propelled in a natural way. If I'm flying or swimming or launching a rocket, all that propulsion is planned.

**Comment by alanruttenberg** [comment]
@neilotte sorry meant to tag @paulnmckelvey who I was responding to.

**Comment by paulnmckelvey** [comment]
@alanruttenberg I believe the argument of "Natural vs. Intentional processes" presents a false dichotomy.  

I would offer Radio Interference for consideration:

"A NATURAL Process in which a radio signal is disrupted, whether UNINTENTIONALLY or as the result of an [INTENTIONAL] Act of Radio Jamming, Act of Radio Spoofing, or similar Planned Act."

I am inferring from this definition that there is no dichotomy between INTENTION and NATURE; that ACTS (intentional or otherwise) can (and most likely, must) utilize or employ NATURAL PROCESSES to achieve their end states or objectives.  

Example.  A pilot operates an airplane (Act of Artifact Employment/Vehicle Use, or Act of Travel) which employs a propeller to realize a Propulsion Process, which overcomes drag and produces horizontal Motion, which results in air flow over an airfoil (wing), which produces lift (vertical Motion) allowing the airplane to fly above the ground.

In this example, ACTS and NATURAL PROCESSES coexist.  And just because an ACT necessarily leads to the initiation of a NATURAL PROCESS does not negate the NATURAL "character" of that PROCESS.

At the risk of overextending my argument, it also seems to me that Acts are necessarily dependent on Natural Processes, but that Natural Processes are not necessarily dependent on Acts (intentional or otherwise).  

Finally, perhaps a more valid dichotomy would be NATURAL PROCESSES vs. "ARTIFICIAL PROCESSES?"

**Comment by alanruttenberg** [comment]
@paulnmckelvey Yes, every process in existence depends on physics. That's kind of what you are saying.  But splitting out the pure "physics" aspects vs the intentional aspect seems unnecessarily painful.  Let's take an example: Act of expressive communication. What part of the process is natural. What part is not natural? And lets say you can do it (I can't)

Say I'm a comedian and for slapstick part of my act is tripping over something silly. I think that that particular tripping is a planned process, but that other trippings are not. When we see a case like this I'd prefer to avoid classifying tripping as either planned or unplanned and instead either assert the planned aspect at the instance level, or have that it is planned inferred by it being the realization of a concretization of a plan.

It's not that there's a clear divide between planned and natural processes It's that there are many processes that because of their type must have been planned and then a whole lot of other processes that might go either way. A rock rolls down a hill because of minor earthquake, or because I intended it and threw it. Both rocks rolling, one planned the other not.

I see many of the natural processes as being of that sort. There is intentional propulsion and accidental propulsion. One shouldn't have to struggle over whether to put it under heading or the other, or even worse duplicate the class with one planned and the other natural.

In developing OBI we had this discussion many years ago and decided it was untenable to make this distinction.
